sarathi/model_executor/attention/flashinfer_attention_wrapper.py

```diff
@@ -46,19 +44,16 @@ def init(
         self.append_kv_page_indices_tensor = None
         self.append_kv_page_indptr_tensor = None
         self.append_kv_last_page_len_tensor = None
-
+        
     def to_int_tensor(self, data: List[int]) -> torch.Tensor:
         return torch.tensor(data, dtype=torch.int32, device="cuda")
 
-    def get_cache_block(self, num_blocks: int, **kwargs) -> torch.Tensor:
-        return torch.randn(
-            num_blocks,
-            2,
-            self.block_size,
-            self.num_kv_heads,
-            self.head_dim,
-            **kwargs,
-        )
+    def get_cache_block_size(self) -> int:
```

Comment:
where is this used now?

Reply:
This is used only within the [model_runner.py](https://github.com/microsoft/sarathi-serve/pull/32/files#diff-1b39c65676e958c0eb071353a4f0aa465da5ebacf279b23f7fe5978ef63db5bb) file